discovery/xds/xds.go

```diff
@@ -154,23 +156,19 @@ func (d *fetchDiscovery) poll(ctx context.Context, ch chan<- []*targetgroup.Grou
 		return
 	}
 
-	parsedGroups, err := d.parseResources(response.Resources, response.TypeUrl)
+	parsedTargets, err := d.parseResources(response.Resources, response.TypeUrl)
 	if err != nil {
 		level.Error(d.logger).Log("msg", "error parsing resources", "err", err)
 		d.fetchFailuresCount.Inc()
 		return
 	}
 
-	for _, group := range parsedGroups {
-		group.Source = d.source
-	}
-
-	level.Debug(d.logger).Log("msg", "updated to version", "version", response.VersionInfo, "groups", len(parsedGroups))
+	level.Debug(d.logger).Log("msg", "updated to version", "version", response.VersionInfo, "targets", len(parsedTargets))
 
 	// Check the context before sending an update on the channel.
```

Comment:
This comment is technically incorrect, can you remove it? Go does not follow the order in which it is written.